src/scripts/components/Play.js

```diff
@@ -72,6 +72,7 @@ const Wrapper = styled.div`
   flex-direction: column;
```

Comment:
Basically all this is is adding a new `div` around the video element; apparently the `padding-bottom` aspect ratio css trick doesn't work in firefox unless you do this